package-lock.json

```diff
@@ -24,8 +24,8 @@
         "@types/node": "^12.0.0",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
-        "clsx": "^1.1.1",
```

Comment:
🤔 Not really a part of this review but food for thought: Is there any way we could automate detection of whether a developer has remembered to commit the updated the lock file when updating `package.json`?